sys/include/xtimer/tick_conversion.h

```diff
@@ -122,6 +123,24 @@ static inline uint64_t _xtimer_usec_from_ticks64(uint64_t ticks) {
     return (usec >> 9); /* equivalent to (usec / 512) */
 }
 
+#elif XTIMER_HZ == MHZ(10)
+
+static inline uint32_t _xtimer_ticks_from_usec(uint32_t usec) {
+    return usec * 10;
+}
+
+static inline uint64_t _xtimer_ticks_from_usec64(uint64_t usec) {
+    return usec * 10;
+}
+
+static inline uint32_t _xtimer_usec_from_ticks(uint32_t ticks) {
+    return ticks / 10;
+}
+
+static inline uint64_t _xtimer_usec_from_ticks64(uint64_t ticks) {
+    return ticks / 10;
```

Comment:
```suggestion
    return (ticks + 5) / 10;
```

does this help with the jitter?

Reply:
I assumed that the test failures are due to that the Linux scheduler is not accurate enough for the millisecond delays. I can try this, but I think it is not going to matter a lot with the timer running at 10 MHz.